src/Illuminate/Routing/Console/ControllerMakeCommand.php

```diff
@@ -268,6 +270,35 @@ protected function generateFormRequests($modelClass, $storeRequestClass, $update
         return [$storeRequestClass, $updateRequestClass];
     }
 
+    /**
+     * Build the return type replacement values.
+     *
+     * @param  array  $replace
+     * @param  string|null  $return
+     * @return array
+     */
+    protected function buildReturnTypeReplacements(array $replace, string|null $returnType)
+    {
+        $returnTypes = match ($returnType) {
+            'response' => ['use Illuminate\Http\Response;', ': Response'],
+            'json' => ['use Illuminate\\Http\\JsonResponse;', ': JsonResponse'],
+            'view' => ['use Illuminate\Contracts\View;', ': View\\View|View\\Factory'],
+            'inertia' => ['use Inertia\Response;', ': Response'],
+            default => ['', ''],
+        };
+
+        $replace = array_merge($replace, [
+            '{{ returnTypeUse }}' => $returnTypes[0],
```

Comment:
This should be returnTypeClass?

Reply:
Fixed that part already, caught by static analysis. There are two variables: returnTypeUse and returnTypeClass.